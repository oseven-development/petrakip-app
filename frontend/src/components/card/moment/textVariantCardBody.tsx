import { IonText } from '@ionic/react'

export interface TextVariantProps {
  variant: 'text'
  text: string
}
export const TextVariantCardBody: React.FC<TextVariantProps> = ({ text }) => (
  <IonText>{text}</IonText>
)
