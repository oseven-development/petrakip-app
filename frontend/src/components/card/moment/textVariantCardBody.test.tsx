import { render } from '@testing-library/react'

import { TextVariantCardBody } from './textVariantCardBody'

describe('TextVariantCard Component', () => {
  const audioVariantCard = (
    <TextVariantCardBody variant="text" text="mySource" />
  )

  test('renders without crashing', () => {
    const { container } = render(audioVariantCard)
    expect(container).toMatchSnapshot()
  })
})
