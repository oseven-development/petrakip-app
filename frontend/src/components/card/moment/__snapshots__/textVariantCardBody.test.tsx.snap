// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextVariantCard Component renders without crashing 1`] = `
<div>
  <ion-text>
    mySource
  </ion-text>
</div>
`;
